Source/ZXing.Net.Mobile.Android/CameraAccess/CameraController.cs

```diff
@@ -232,29 +251,65 @@ private void ApplyCameraSettings()
             var supportedFocusModes = parameters.SupportedFocusModes;
             if (_scannerHost.ScanningOptions.DisableAutofocus)
                 parameters.FocusMode = Camera.Parameters.FocusModeFixed;
+            else if (supportedFocusModes.Contains(Camera.Parameters.FocusModeAuto))
+                parameters.FocusMode = Camera.Parameters.FocusModeAuto;
             else if (Build.VERSION.SdkInt >= BuildVersionCodes.IceCreamSandwich &&
                 supportedFocusModes.Contains(Camera.Parameters.FocusModeContinuousPicture))
                 parameters.FocusMode = Camera.Parameters.FocusModeContinuousPicture;
             else if (supportedFocusModes.Contains(Camera.Parameters.FocusModeContinuousVideo))
                 parameters.FocusMode = Camera.Parameters.FocusModeContinuousVideo;
-            else if (supportedFocusModes.Contains(Camera.Parameters.FocusModeAuto))
-                parameters.FocusMode = Camera.Parameters.FocusModeAuto;
             else if (supportedFocusModes.Contains(Camera.Parameters.FocusModeFixed))
                 parameters.FocusMode = Camera.Parameters.FocusModeFixed;
```

Comment:
Changing order of focus selection has completely broke scanner autofocus on my device (Vernee thor).